src/rocm_docs/rocm_docs_theme/components/left-side-menu.html

```diff
@@ -0,0 +1,3 @@
+<a class="navbar-brand logo" href="https://rocm.docs.amd.com/en/latest/">
```

Comment:
we need this to be dynamic based on branch

Reply:
Done in ccda756